kubectl-rook-ceph.sh

```diff
@@ -230,20 +240,20 @@ function rook_version() {
 
 function run_rook_version() {
   end_of_command_parsing "$@" # end of command tree
-  $TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" exec deploy/rook-ceph-operator -- rook version
+  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-operator -- rook version
 }
 
 function run_rook_cr_status() {
   if [ "$#" -eq 1 ] && [ "$1" = "all" ]; then
-    cr_list=$($TOP_LEVEL_COMMAND --namespace "$ROOK_CLUSTER_NAMESPACE" get crd | awk '{print $1}' | sed '1d')
+    cr_list=$(KUBECTL_NS_CLUSTER get crd | awk '{print $1}' | sed '1d')
```

Comment:
@parth-gr and @subhamkrai this is also an example of how a function can be used in the same way as a command.

If this were my project, I would probably have a lot of these smaller functions that repeat commonly-repeated commands for kubectl. For example, I could also have a function like below for executing commands in the operator.

```sh
function exec_in_operator() {
  KUBECTL_NS_OPERATOR exec deploy/rook-ceph-operator -- "$@"
}
```

Some reasons to do this:
- because code is reused, if I want to change how commands are issued, I can change fewer lines of code
- code might be easier to read
- the important parts of code lines (the command being executed) is less likely to be pushed off to the right side of the screen (easier to see)

Some reasons not to do this:
- while code could be easier to read, that is subjective; it could be harder to read if the function name isn't very clear

Do you think the benefits of this PR outweigh the drawbacks? Is this a pattern you would like to see used in the project? Since I see @subhamkrai as the primary maintainer of this utility, I think this is a good thing for him to decide. What do you think?

Reply:
It looks good to me, by following this approach.

Reply:
Looks more readable and cleaner to me. Thanks!